Superpill (Pill-Class derivative)

HanSolo1519 opened the pull request:
## About The Pull Request

Adds the _'Pill Class-R(adioactive) Execution Device'_, a long-ranged SM-powered modification to the well-known and beloved Pill class pod.

![superpill](https://user-images.githubusercontent.com/72716882/168697230-48aedc03-c92b-4a04-a823-825f5d1069cd.gif)

Video proof of function/tutorial
[![Superpill](http://img.youtube.com/vi/nN5oRB5wMtE/0.jpg)](http://www.youtube.com/watch?v=nN5oRB5wMtE "Superpill")

## Why It's Good For The Game

Adds a small ship with a comparable size to the sugarcube (4x3 vs 6x12, only an 83% difference) with an interesting engine mechanic.


## Changelog
:cl:
add: Superpill
change: Stopped the sneaky little git of a crystal from sliding under the glass level
/:cl:

HanSolo1519 committed:
Add files via upload

tiramisuapimancer commented:
what in gods name

PiperDoots commented:
This is so terrible, I love it

rye-rice commented:
And i thought I peaked with the pill C

Latentish commented:
Please add this as an adminspawn only for the love of god.

MusselMain commented:
As a fan of the Pill class series, I really love this, and wish that all the pills come back either someday in the future, or as a shits and giggles thing every couple of weeks

Fikou commented:
u have to make the supermatter layer above windows so it doesnt look like shit

HanSolo1519 committed:
Add files via upload

HanSolo1519 commented:
Should be fixed now
![image](https://user-images.githubusercontent.com/72716882/168804722-27b1c1dc-cb46-4af0-ad9c-ee1546587909.png)

Vexylius commented:
As impressive of an engineering feat as this is, I am **not** adding this to regular map rotation, maybe as an admin-only spawn

HanSolo1519 commented:
> As impressive of an engineering feat as this is, I am **not** adding this to regular map rotation, maybe as an admin-only spawn

![ezgif-5-f739eaa930](https://user-images.githubusercontent.com/72716882/169089485-995803ae-99cb-4af3-bb2a-517efd93e152.gif)

Honestly I would be surprised otherwise, the normal Pill and Independant Pill are both adminspawns anyway, it would be odd to put the "Cancerous PIll" in rotation

Fikou commented on _maps/shuttles/shiptest/pill_super.dmm:
```diff
@@ -0,0 +1,290 @@
+/area/ship/storage)
+"i" = (
+/obj/machinery/power/supermatter_crystal{
+	layer = 3.4
```
could you make this a code change 🥺

HanSolo1519 replied:
You just want me to make another pr that puts all sms on the 3.4 layer by default?

ZephyrTFA replied:
Use the shard subtype instead of the base crystal

HanSolo1519 replied:
Replaced the crystal with a shard, still gotta make the higher layer for the SM it's own PR

Also turns out that since I moved from the ghetto invisible lattice thing for the windows on other tiles and moved them to the same tile as the SM, my precious unreinforced glass explodes, I replaced them with cool purple windows to fix this.

HanSolo1519 committed:
Replace windows and sm with pwindows and sm shard

Made the windows not explode, dechadified the SM by replacing the crystal with a shard

Calyxman commented:
Needs a proper laser gun, extra cells, antirads and radsuits.
Also a bike horn
